Eric-Arellano/ngsc-service-hours
# #11 Display No-Show Data

# Objective
Pull and display no-show data after students log in.
Based on Issue #11.

# Changes Made
- [X] Added API Call to pull no-show data from the master spreadsheet
- [X] Added section to engagement view to display attendance data
- [X] Added to `EngagementViewContainer` to pass on no-show data

# How to Test
1. Checkout the `show-no-shows` branch
2. Run the scripts `run-backend.sh` and `run-frontend.sh`
3. Type in a valid student ID into the input
4. Verify you are the person attached to the ID entered
5. See that there is now an Attendance section that displays the number of no-shows for the person.

# Possible Regressions
﻿`EngagementContainerView` now has to run through a chain of 2 asynchronous calls, which could be problematic to consider later on. No regressions expected, though.

---
CC: @Eric-Arellano
